CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
+* Changed parsed queries using the `between` operator to be inclusive of the limits, a closed interval instead of an open interval. This is to conform to the published documentation and for parity with NSPredicate's definition. ([#5262](https://github.com/realm/realm-core/issues/5262), since the introduction of this operator in v11.3.0)
```

Comment:
This line was in the wrong section due to a previous incorrect merge that I noticed while rebasing this commit.

Reply:
I'm not sure what's going on with the merge, but I am pretty sure the fix that this message is referring to did make it into the v11.10.0 release. At least the commit is included in the v11.10.0 [tag](https://github.com/realm/realm-core/blob/v11.10.0/CHANGELOG.md).

Reply:
Oh, I confused myself then and it was correct previously.